x-pack/plugins/security_solution/server/lib/prebuilt_saved_objects/mocks.ts

```diff
@@ -328,7 +328,7 @@ export const expectedSavedObjectTemplate = [
                     operationType: 'last_value',
                     params: { sortField: '@timestamp' },
                     scale: 'ordinal',
-                    sourceField: 'risk',
+                    sourceField: 'risk.keyword',
```

Comment:
Is it intentional?

Reply:
yeah, otherwise the aggregation wouldn't be displayed properly in dashboard, as text field can't be aggregated.